packages/@lwc/integration-karma/test/component/LightningElement.attachInternals/api/index.spec.js

```diff
@@ -39,7 +39,7 @@ const attachInternalsSanityTest = (tagName, ctor) => {
         });
 
         afterAll(() => {
-            document.body.removeChild(elm);
+            elm.remove();
```

Comment:
IIRC, `elm` is not always a child of `document.body`, so doing that invocation would fail, while this removes `elm` from whichever parent it has.